Gruntfile.js

```diff
@@ -155,7 +111,79 @@ module.exports = function(grunt) {
       }
     },
 
+    /**
+     * Build JavaScript with Webpack.
+     */
+    webpack: {
+      options: {
+        entry: './js/index.js',
+        output: {
+          filename: 'dist/neue.js',
+          library: 'Neue',
+          libraryTarget: 'umd'
+        },
+        externals: {
+          // Don't bundle the 'jquery' package in neue.js, but
+          // instead load from `jQuery` global variable or AMD/CJS package.
+          'jquery': {
+            root: 'jQuery',
+            commonjs: 'jquery',
+            commonjs2: 'jquery',
```

Comment:
See [docs](http://webpack.github.io/docs/configuration.html#output-librarytarget)! Actually just realized I can safely remove plain-ol' `commonjs` from this object too – `umd` exports to AMD, CommonJS2 or as property in root.